docs/equinix/index.md

```diff
@@ -201,6 +200,10 @@ helm -n metallb-system install metallb metallb/metallb \
 
 ## Installing Sextant on Equinix Metal
 
+### <a name="sign-up"></a>Sign up for Evaluation
```

Comment:
in-line HTML is bad. In particular here since all headings automatically turn into anchors.

Reply:
How am I supposed to point someone at this section if they sailed past it and are now trying to create secret without credentials? Or is that too helpful :)